src/rws_poco_client.cpp

```diff
@@ -333,7 +333,7 @@ POCOClient::POCOResult POCOClient::webSocketConnect(const std::string uri, const
   {
     result.addHTTPRequestInfo(request);
     p_websocket_ = new WebSocket(client_session_, request, response);
-    p_websocket_->setReceiveTimeout(Poco::Timespan(60000000));
+    p_websocket_->setReceiveTimeout(Poco::Timespan(LONG_TIMEOUT));
```

Comment:
`LONG_TIMEOUT`: name is a bit random ;)

Reply:
Well, not too random I think since it's a longer timeout than the default timeout. Maybe the timeout should be made configurable in the future.

Reply:
I guess my comment was more about why you chose this name. `LONG_TIMEOUT` is arbitrary, as the name does not convey anything about *how long* the timeout is. Reading your code I cannot deduce how long things will wait before timing out unless I go and lookup the declaration of the constant.